mod/pages/views/default/resources/pages/edit.php

```diff
@@ -30,15 +29,7 @@
 
 if ($page->canEdit()) {
 
-	if ($revision) {
-		$revision = elgg_get_annotation_from_id($revision);
-		if (!$revision || !($revision->entity_guid == $page_guid)) {
-			register_error(elgg_echo('pages:revision:not_found'));
-			forward(REFERER);
-		}
-	}
```

Comment:
We no longer allow editing specific revisions. At least there's UI that leads to it.